Slim/Formats/Playlists/CUE.pm

```diff
@@ -16,6 +16,8 @@ use File::Slurp;
 use File::Spec::Functions qw(catdir);
 use Scalar::Util qw(blessed);
 
+use Data::Dump qw(dump); #could be removed in production, usefull in debug.
+
```

Comment:
Could you please get rid of the commented out debug logging and use LMS' log infrastructure instead? Don't import dump(), but call something like:

```perl
main::DEBUGLOG && $log->is_debug && $log->debug(Data::Dump::dump({...}));
```

This not only makes the code cleaner, but it also allows to enable/disable logging at run time. The two conditions make sure the rather expensive dump() call is only executed when desired.